.devcontainer/README.md

```diff
@@ -0,0 +1,7 @@
+# Overview
+
+Do not use a default configuration when starting a new devcontainer.
```

Comment:
```suggestion
Do not use the default configuration when starting a new devcontainer.
```

Reply:
actually I do not know what is default, it is something which we should never run.